src/pages/Dragons.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from 'react';
+import { connect } from 'react-redux';
+import store from '../redux/configureStore';
+import Displayer from '../components/Displayer';
+import { getDragons } from '../redux/api/api';
+
+function Dragons(props) {
+  const [dragonsDisplay, setDragonsDisplay] = useState(null);
+  const [calledDragon, setCalledDragon] = useState(null);
+
+  useEffect(() => {
+    const { dragons } = props;
```

Comment:
- [ ] Rather than this kindly use the useSelector hook from redux to retrieve the value of data in the store. Check the specific project requirement [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md#:~:text=Use%20useSelector()%20Redux%20Hook%20to%20select%20the%20state%20slices%20and%20render%20lists%20of%20rockets%20and%20missions%20in%20corresponding%20routes.%20i.e.%3A).

Reply:
Thank you, I'll get right to it.